README.md

```diff
@@ -88,6 +88,36 @@ while (accounts.hasMore()) {
 }
 ```
 
+#### Additional Pager Methods
+
+In addition to the methods to facilitate pagination, the Pager class provides 2 helper methods:
+
+1. getFirst
+2. getCount
+
+##### First
+
+The Pager's `getFirst` method can be used to fetch only the first resource from the endpoint.
+
+```java
+Pager<Account> accounts = client.listAccounts(new QueryParams());
+Account account = accounts.getFirst();
+System.out.println(account.getCode());
+```
+
+##### Count
+
+The Pager's `getCount` method will return the total number of resources that are available at the requested endpoint.
+
+```java
+Pager<Account> accounts = client.listAccounts(new QueryParams());
+int total = accounts.getCount();
+System.out.println("There are " + total + " accounts in total");
+for (Account account : accounts) {
```

Comment:
Should we remove this part? I worry it people might expect the number of accounts to come out of here to be equal to the `total`. I also think we could reword the description to be clear that it counts the total that match the `QueryParams` at the time that `getCount` is called.